### PR TITLE
DOCSP-32400 add Link to the ADF query limits page from the Atlas CLI docs

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
## Proposed changes

Adds Link to the ADF query limits page from the Atlas CLI

_Jira ticket:_ DOCSP-32400

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by sarahsimpers — DISMISSED** [review]
LGTM

File: internal/cli/datafederation/querylimits/create.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func CreateBuilder() *cobra.Command {
 	opts := &CreateOpts{}
 	cmd := &cobra.Command{
 		Use:   "create <name>",
-		Short: "Creates a new Data Federation query limit.",
+		Short: "Creates a new Data Federation query limit. To learn more about the query limits, see https://dochub.mongodb.org/core/adf-query-limits.",
```
Comment (gssbzn) [inline internal/cli/datafederation/querylimits/create.go:81]: can you please move it to the long description?

File: internal/cli/datafederation/querylimits/create.go [hunk 2]
```diff
@@ -78,7 +78,7 @@ func CreateBuilder() *cobra.Command {
...
 		Long:  fmt.Sprintf(usage.RequiredRole, "Project Owner"),
```
Comment (gssbzn) [inline internal/cli/datafederation/querylimits/create.go:82]:
```suggestion
		Long:  To learn more about the query limits, see https://dochub.mongodb.org/core/adf-query-limits." + fmt.Sprintf(usage.RequiredRole, "Project Owner"),
```

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**